spec/alavetelitheme_spec.rb

```diff
@@ -0,0 +1,11 @@
+# If defined, ALAVETELI_TEST_THEME will be loaded in config/initializers/theme_loader
+ALAVETELI_TEST_THEME = 'alavetelitheme'
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','..','..','spec','spec_helper'))
```

Comment:
This setup should probably be in a spec_helper actually https://github.com/mysociety/derechoapreguntar-theme/blob/master/spec/spec_helper.rb